services/brig/src/Brig/API/Connection.hs

```diff
@@ -1,6 +1,10 @@
 {-# LANGUAGE RecordWildCards   #-}
 
 -- TODO: Move to Brig.User.Connection (& split out Brig.User.Invitation?)
+
+-- | > docs/reference/user/connection.md {#RefConnection}
```

Comment:
I decided to add file names where feasible, because then it's easy to jump to them right from the editor (in Emacs: [`M-x projectile-find-file-dwim`](https://projectile.readthedocs.io/en/latest/configuration/#projectile-find-file-dwim)).

The `>` is there to stop Haddock from screwing up the formatting by treating slashes as italics.